back-end/services/user.py

```diff
@@ -82,21 +102,21 @@ def login(username, password):
             cursor.execute(query, (username, hashpass))
             result = cursor.fetchall()
             if (not result):
-                response = {"message": "Invalid credentials"}
+                response = {"message": "Invalid credentials"}, 400
                 cnx.close()
-                return response, 200
-                
+                return response, 400      
             else:
-                response = {"message": "Logged in","user_id":result[0][0]}
-                cnx.close()
-                return response, 200
-                
+                response = {"message": "Logged in"}, 200
+                cnx.close()   
+                return response, 200   
         except Error as e:
             print("Error while connecting to MySQL", e)
             #log if any errors happen with connection
-            response = {"message": "Error while connecting to MySQL"}
+            response = {"message": "Error while connecting to MySQL"}, 500
             cnx.close()
             return response, 500
```

Comment:
```suggestion
            return response
```

Reply:
response already has response code, so can we get rid of it here. I think a lot of end points have multiple response codes in them so may be a refactor thing.